web/app.js

```diff
@@ -718,7 +718,7 @@ const PDFViewerApplication = {
   },
 
   get loadingBar() {
-    const bar = new ProgressBar("#loadingBar");
+    const bar = new ProgressBar("loadingBar");
     return shadow(this, "loadingBar", bar);
```

Comment:
This is unrelated to this patch, does it make sense to shadow the value ?
I suppose that once the document is loaded we don't need it anymore.

Reply:
> This is unrelated to this patch, does it make sense to shadow the value ?

Doing that simplifies the code overall, and also allows us to initialize it *lazily*, since we're accessing it from multiple spots throughout the viewer code; see e.g.
 - https://github.com/mozilla/pdf.js/blob/13c01b6d4ababf039fe59ef800ebae38b1de37f5/web/app.js#L1163-L1164
 - https://github.com/mozilla/pdf.js/blob/13c01b6d4ababf039fe59ef800ebae38b1de37f5/web/app.js#L1180-L1183
 - https://github.com/mozilla/pdf.js/blob/13c01b6d4ababf039fe59ef800ebae38b1de37f5/web/app.js#L1196
 - https://github.com/mozilla/pdf.js/blob/13c01b6d4ababf039fe59ef800ebae38b1de37f5/web/app.js#L1255

> I suppose that once the document is loaded we don't need it anymore.

Correct; however if we'd remove it at the end of loading that'd (as far as I can tell) either mean having to initialize it eagerly or possibly require adding more code.